src/GameLogic/PlayerActions/Party/PartyRequestAction.cs

```diff
@@ -20,26 +20,29 @@ public class PartyRequestAction
     /// <param name="toRequest">The player which receives the request.</param>
     public async ValueTask HandlePartyRequestAsync(Player player, Player toRequest)
     {
+        var playerInParty = player.Party != null && !Equals(player.Party.PartyMaster, player);
```

Comment:
The variable name is misleading. It also checks if the player is the party master.
Suggestion: isPartyMember